examples/generics/3_media.py

```diff
@@ -11,10 +23,16 @@
 from dff.utils.testing.common import check_happy_path, is_interactive_mode, run_interactive_mode
 from dff.utils.testing.response_comparers import generics_comparer
 
-kitten_id = "Y0WXj3xqJz0"
-kitten_ixid = "MnwxMjA3fDB8MXxhbGx8fHx8fHx8fHwxNjY4NjA2NTI0"
-kitten_width = 640
-kitten_url = f"https://unsplash.com/photos/{kitten_id}/download?ixid={kitten_ixid}&force=true&w={kitten_width}"
+
+# %%
+kitten_id = "Y0WXj3xqJz0"  # id of the picture
+kitten_ixid = "MnwxMjA3fDB8MXxhbGx8fHx8fHx8fHwxNjY4NjA2NTI0"  # ixid of the picture
+kitten_width = 640  # Width of the picture
+kitten_url = (  # URL of the picture with a kitten.
```

Comment:
Those comments are redundant.
```suggestion
# %%
# kitten picture info:
kitten_id = "Y0WXj3xqJz0"
kitten_ixid = "MnwxMjA3fDB8MXxhbGx8fHx8fHx8fHwxNjY4NjA2NTI0"
kitten_width = 640
kitten_url = (
```

Reply:
There were two requests for them from @kudep

Reply:
Yes, I saw those requests. But the comments are still redundant -- they just append "of the picture" to the second part of variable's name. I think it is better to add a single comment explaining the purpose of those variables as a whole rather than adding a comment to each of the variables.